package.json

```diff
@@ -31,5 +31,10 @@
     "supertest": "^4.0.2",
     "tap-spec": "^5.0.0",
     "tape": "^4.11.0"
-  }
+  },
+  "standardx": {
+    "ignore": [
+      "lib/context.js"
```

Comment:
Is this because standardx doesn't grok the `#response` syntax?

Reply:
Yeah, it will complain about that and I meant to ask you if it is alright to use private members.

Reply:
I think it's fine to use private members since we will be controlling the version of node.js in use.